script.js

```diff
@@ -0,0 +1,79 @@
+const h2El = document.querySelector('h2');
```

Comment:
normally, yes h2El would probably apt, but here we require context as to why we are using this el, so displayEl or displayContainer or somehting that gives a future dfev the context as to why we are using this ui var